docs/introduction/infrastructure.md

```diff
@@ -1,161 +1,104 @@
 # Computational Infrastructure
 
-## Introduction
-One way of understanding the Inductiva API is to see it as an abstraction layer that
-allows you to use computational resources made available by different providers in a
-seamless and uniform fashion. 
+One way to understand the Inductiva API is to see it as an intermediary between 
+users and the complex landscape of computational resources. In this reference, you 
+will learn how the Inductiva API manages your computational workload, and then you'll
```

Comment:
Not sire reference is a good pointer here. I think you don't to keep repeating `section` which is fair, but reference seems to point to other material.